lib/util/ws/notify/order_executed.js

```diff
@@ -3,14 +3,16 @@
 const _flatten = require('lodash/flatten')
 const _isFinite = require('lodash/isFinite')
 const _capitalize = require('lodash/capitalize')
+const _lowerCase = require('lodash/lowerCase')
 const notifySuccess = require('./success')
 
 module.exports = (ws, exID, order) => {
-  const { amount, symbol, price, type } = order
+  const { id, amount, symbol, price, type } = order
 
   notifySuccess(ws, _flatten([
-    'Executed', _capitalize(exID), type, amount > 0 ? 'BUY' : 'SELL',
-    'order on', `${symbol}`,
-    _isFinite(+amount) && _isFinite(+price) && +price > 0 && [+amount, '@', price]
+    _capitalize(exID), _lowerCase(type), amount > 0 ? 'BUY' : 'SELL',
```

Comment:
can you use `originalAmount` to check for `BUY` or `SELL` order

Reply:
@avsek477 how to reproduce these cases?

Reply:
partially filled events will be useful. try fulfilling orders with half amount ..  below is the order details array for partially filled event where 0.009 is the remaining amount and 0.001 is the filled amount.. where the original amount is 0.01
```
[
  1257243020,
  null,
  1628499046212,
  'tBTCUSD',
  1628499078210,
  0.009,
  0.01,
  'EXCHANGE LIMIT',
  'PARTIALLY FILLED @ 47000.0(0.001)',
  47000
]
```

Reply:
https://github.com/bitfinexcom/bfx-hf-server/pull/109#discussion_r685844500